bundles/org.openhab.binding.tidal/README.md

```diff
@@ -0,0 +1,293 @@
+The binding will show them as _GONE_.
+
+## Discovery
+
+As long as Tidal Connect devices are available in the context of the user account configured with the bridge/bridges they should show up whenever you initiate discovery of things.
+
+If no devices are showing up, try to connect to the device(s) from your smartphone or computer to make sure the device(s) are in use by your user account.
+
+The discovery of devices in the Tidal Web API is based on what is known by Tidal.
+There is difference between e.g. smartphones and computers which can discover devices on the local network and the Web API which is not able to do so.
+It only knows about a device if your account is currently associated with the device.
+
+## Channels
+
+### Bridge / Player
+
+The channels on the bridge are the ones used to both control the active device and get details of currently playing music on the Tidal Account associated with the bridge.
+
+**Common Channels:**
+
+| Channel Type ID | Item Type | Read/Write | Description                                                                                       |
+| --------------- | --------- | ---------- | ------------------------------------------------------------------------------------------------- |
+| deviceName      | String    | Read-write | Name of the currently active Connect Device,                                                      |
+| devices         | Selection | Read-write | List of currently active Connect Devices, Set the device ID to transfer play to that device.      |
+| deviceVolume    | Dimmer    | Read-write | Get or set the active Connect Device volume.                                                      |
+| deviceShuffle   | Switch    | Read-write | Turn on/off shuffle play on the active device.                                                    |
+| trackPlay       | String    | Read-write | Set which music  to play on the active device. This channel accepts Tidal URIs and URLs.        |
+| trackPlayer     | Player    | Read-write | The Player Control of the active device. Accepts PLAY/PAUSE/NEXT/PREVIOUS commands.               |
+| trackRepeat     | String    | Read-only  | `track` repeats the current track. `context` repeats the current context. `off` turns repeat off. |
+| trackName       | String    | Read-only  | The name of the currently playing track.                                                          |
+| trackDuration   | String    | Read-only  | The duration (m:ss) of the currently playing track. This is updated every second.                 |
+| trackDurationMs | Number    | Read-only  | The duration of the currently playing track in milliseconds.                                      |
+| trackProgress   | String    | Read-only  | The progress (m:ss) of the currently playing track. This is updated every second.                 |
+| trackProgressMs | Number    | Read-only  | The progress of the currently playing track in milliseconds.                                      |
```

Comment:
Did you consider this to be a `Number:Duration` type. (same for the other Ms channels)